target/ppc/translate.c

```diff
@@ -4520,6 +4520,20 @@ static void gen_sc(DisasContext *ctx)
     uint32_t lev;
 
     lev = (ctx->opcode >> 5) & 0x7F;
+
+#if defined(TARGET_PPC64) && !defined(CONFIG_USER_ONLY)
+    /*
+     * The PowerPC semihosting specifies the following
```

Comment:
<img src="https://render.githubusercontent.com/render/math?math=\lim_{nit%20\to%20\infinity}">: "specifies" may not be the best word since ppc doesn't have a semihosting specification

Reply:
Right, fixed.